fe/src/components/PdfDropzone/PdfDropzone.tsx

```diff
@@ -0,0 +1,98 @@
+"use client";
+
+import { Button } from "@/lib/materialTailwindExports";
+import { FormEvent, useState } from "react";
+import { useDropzone, FileWithPath } from "react-dropzone";
+import { uploadPdfToApi } from "./uploadPdfToApi";
+import toast from "react-hot-toast/headless";
+import { useRouter } from "next/navigation";
+
+// TODO: not allow this page access if submission status is pending
+function PdfDropzone() {
+  const maxFileSize = 10485760;
+  const [uploading, setUploading] = useState(false);
```

Comment:
```suggestion
  const [isUploading, setIsUploading] = useState(false);
```
More common way to name state variables.